README.md

```diff
@@ -72,7 +72,7 @@ function App() {
       <button
         onClick={async () =>
           open({
-            integrationId: IntegrationIds.Salesforce,
+            integrationId: 'salesforce',
```

Comment:
I don't have much context around this, but could you explain why we're switching away from enums?

Reply:
Ah we decided not to release an Enum in the types of the react-vessel-link package because we want people to set up their FEs in such a way that it dynamically reacts to new integrations we support (via the /crm/integrations endpoint)